### PR TITLE
created color pallete info page

### PR DESCRIPTION
**Pull request by kbojarowska**

### CODE REVIEW

**Comment by dpionk** [comment]
I think we should have one component called Pallet Test with concrete background and display components in it.

**Comment by dpionk** [comment]
Or we should create the default background template

File: skin-resque-frontend/src/App.js [hunk 1]
```diff
@@ -10,6 +10,7 @@ import FrontPage from './pages/FrontPage/FrontPage';
 import CosmeticsCatalogue from './pages/Cosmetics/CosmeticsCatalogue/CosmeticsCatalogue';
 import RecipeDetails from './pages/Cosmetics/RecipeDetails/RecipeDetails'
 import SkintypeTestPage from './pages/SkintypeTest/SkintypeTestPage/SkintypeTestPage'
+import Info from './pages/ColorPallete/Info';
```
Comment (MrSpaceMan1) [inline skin-resque-frontend/src/App.js:13]: typo, it's Palette

File: skin-resque-frontend/src/pages/ColorPallete/Info.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
+import { Link } from 'react-router-dom';
+import { Button, Heading, Text } from "../../components"
+import './Info.scss'
+
+function Info() {
+  return (
+	<div className='page' style={{ backgroundImage: `url('${process.env.PUBLIC_URL}/images/bg-cosmetics.svg')` }}>
```
Comment (MrSpaceMan1) [inline skin-resque-frontend/src/pages/ColorPallete/Info.js:7]: Background doesn't load on first view of the page. Besides that `/images/...` should be enough

File: skin-resque-frontend/src/pages/ColorPallete/Info.js [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+            <Heading className="heading">Color pallete finder</Heading>
+        <div className="beige-bg">
+            <Text className="txt">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt
```
Comment (MrSpaceMan1) [inline skin-resque-frontend/src/pages/ColorPallete/Info.js:10]: Is your prettier working? Most of this file was un-prettified.

**Review by MrSpaceMan1 — CHANGES_REQUESTED** [review]
Change spelling of `Pallete` to `Palette`

File: skin-resque-frontend/src/pages/ColorPallete/Info.js (hunk 2 above)
Comment (dpionk) [reply skin-resque-frontend/src/pages/ColorPallete/Info.js:7]: I think it should be in scss file as well

Comment (kbojarowska) [reply skin-resque-frontend/src/pages/ColorPallete/Info.js:7]: > Background doesn't load on first view of the page. Besides that `/images/...` should be enough

`/images/...` isnt enough, that was the only way It worked for me

Comment (kbojarowska) [reply skin-resque-frontend/src/pages/ColorPallete/Info.js:7]: > I think it should be in scss file as well

Can you show me how to do it? This was the only way it was working for me

**Review by dpionk — APPROVED** [review]

File: skin-resque-frontend/src/pages/ColorPallete/Info.js (hunk 2 above)
Comment (dpionk) [reply skin-resque-frontend/src/pages/ColorPallete/Info.js]: image

**Review by MrSpaceMan1 — APPROVED** [review]

**Pull request merged**